operator/controller/src/main/java/io/apicurio/registry/operator/util/IngressUtil.java

```diff
@@ -0,0 +1,34 @@
+package io.apicurio.registry.operator.util;
+
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.networking.v1.HTTPIngressPath;
+import io.fabric8.kubernetes.api.model.networking.v1.Ingress;
+import io.fabric8.kubernetes.api.model.networking.v1.IngressRule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Consumer;
+
+public class IngressUtil {
+
+    private static final Logger log = LoggerFactory.getLogger(IngressUtil.class);
+
+    public static String getHost(String component, ApicurioRegistry3 p) {
+        var prefix = p.getMetadata().getName() + "-" + component + "." + p.getMetadata().getNamespace();
+        String host = prefix + ".cluster.example";
```

Comment:
Please, get the suffix from the CR with a default to `.cluster.local` I think.

Reply:
I've seen that domain being used, but it seems like it's reserved for another purpose (Multicast DNS) https://en.wikipedia.org/wiki/Special-use_domain_name .

Reply:
Is that applicable here?

Reply:
This is the hostname as seen from outside of the cluster, correct?
Would it work with a blank suffix to start with?

Reply:
yes. Sure I can do that.